### PR TITLE
NXP-21383: update MP to follow core / JSF split

### PR DESCRIPTION
**Pull request by mincong-h**
I'd like have some feedback about:

- The modification in file `.marketplace/src/main/assemble/assembly.xml`. I'm not sure about the workflow.

### CODE REVIEW

File: marketplace/src/main/assemble/assembly.xml [hunk 1]
```diff
@@ -81,10 +124,14 @@
                 file2="${outdir}/nuxeo-nxr-server-jsf-ui-${nuxeo.distribution.version}.zip"
                 includesfile="${outdir}/includes"
                 excludesfile="${outdir}/excludesfile"
-                patternsetid="agenda.versus.jsf">
+                patternsetid="agenda.versus.serverAndJsf">
```
Comment (troger) [inline marketplace/src/main/assemble/assembly.xml]: Keep `agenda.versus.jsf` for consistency, that's what we did in other packages.

**Review by troger — COMMENTED** [review]
Note that this is not a release commit, so you should rewrite your commit message to something more meaningful of what you did, such as "Update Agenda MP to follow core / JSF split".

Now that you are working on the MP, you could also remove the `ftest`profile in the parent pom, and move the `ftest/webdriver` module to the default `modules` list.

:+1: after fixing my comments :)

File: marketplace/src/main/assemble/assembly.xml [hunk 2]
```diff
@@ -53,26 +54,68 @@
...
           <artifact groupId="org.nuxeo.lib*" />
         </includes>
         <excludes>
           <artifact scope="test" />
         </excludes>
-      </artifact:set>
+      </artifact:dependencies>
     </copy>
     <nx:rmdups dir="${outdir}/nxr/lib" />
+    <!-- TODO What should be unzipped / zipped before `nx:zipdiff`? Not sure what I did is correct. -->
```
Comment (jcarsique) [inline marketplace/src/main/assemble/assembly.xml]: (always associate a TODO with a JIRA ticket: `<!-- TODO NXP-21383 ... -->`)

What you did looks fine.
`nx:zipdiff` must be called when the target NXR is ready. The task will compare the zipped NXR with the server one in order to embed only the missing files in the package.

**Review by jcarsique — APPROVED** [review]
:+1:

**Comment by mincong-h** [comment]
> Now that you are working on the MP, you could also remove the ftestprofile in the parent pom, and move the ftest/webdriver module to the default modules list.

Making `ftest/webdriver` as a default Maven module hooks the functional tests to `mvn verify`, so tests are required before installation: which was previously optional when using `<profile>`. Am I right?

File: marketplace/src/main/assemble/assembly.xml (hunk 2 above)
Comment (mincong-h) [reply marketplace/src/main/assemble/assembly.xml]: Associating a _TODO_ with a JIRA ticket helps to identify to which issue this task belongs. This is a good practice. :+1:

Thanks for this hint and the explanation!

**Review by akervern — APPROVED** [review]

**Comment by akervern** [comment]
👍

**Pull request merged**